doc/mgr/orchestrator.rst

```diff
@@ -88,58 +88,134 @@ filtered to a particular host:
+    node2   /dev/vdc  hdd   8192M  228094  False  LVM detected, Insufficient space (<5GB) on vgs, locked
 
-    Host 192.168.121.181:
-    Device Path           Type       Size    Rotates  Available Model
-    /dev/sdb               hdd      50.0G       True       True ATA/QEMU HARDDISK
-    /dev/sda               hdd      50.0G       True      False ATA/QEMU HARDDISK
 
 .. note::
-    Output form Ansible orchestrator
+    Output form Cephadm orchestrator
 
 Create OSDs
 -----------
 
 Create OSDs on a group of devices on a single host::
 
-    ceph orch osd create <host>:<drive>
-    ceph orch osd create -i <path-to-drive-group.json>
+    ceph orch daemon add osd <host>:device1,device2
 
+or::
 
-The output of ``osd create`` is not specified and may vary between orchestrator backends.
+    ceph orch apply osd -i <json_file/yaml_file> [--preview]
 
-Where ``drive.group.json`` is a JSON file containing the fields defined in
-:class:`ceph.deployment_utils.drive_group.DriveGroupSpec`
+
+or::
+
+    ceph orch apply osd --use-all-devices [--preview]
+
+
+For a more in-depth guide to DriveGroups please refer to :ref:`drivegroups`
+
+Example::
+
+    # ceph orch daemon add osd node1:/dev/vdd
+    Created osd(s) 6 on host 'node1'
+
+
+If the 'apply' method is used. You will be presented with a preview of what will happen.
 
 Example::
 
-    # ceph orch osd create 192.168.121.206:/dev/sdc
-    {"status": "OK", "msg": "", "data": {"event": "playbook_on_stats", "uuid": "7082f3ba-f5b7-4b7c-9477-e74ca918afcb", "stdout": "\r\nPLAY RECAP *********************************************************************\r\n192.168.121.206            : ok=96   changed=3    unreachable=0    failed=0   \r\n", "counter": 932, "pid": 10294, "created": "2019-05-28T22:22:58.527821", "end_line": 1170, "runner_ident": "083cad3c-8197-11e9-b07a-2016b900e38f", "start_line": 1166, "event_data": {"ignored": 0, "skipped": {"192.168.121.206": 186}, "ok": {"192.168.121.206": 96}, "artifact_data": {}, "rescued": 0, "changed": {"192.168.121.206": 3}, "pid": 10294, "dark": {}, "playbook_uuid": "409364a6-9d49-4e44-8b7b-c28e5b3adf89", "playbook": "add-osd.yml", "failures": {}, "processed": {"192.168.121.206": 1}}, "parent_uuid": "409364a6-9d49-4e44-8b7b-c28e5b3adf89"}}
+    # ceph orch apply osd --all-available-devices
+    NAME                  HOST  DATA     DB WAL
+    all-available-devices node1 /dev/vdb -  -
+    all-available-devices node2 /dev/vdc -  -
+    all-available-devices node3 /dev/vdd -  -
+
 
 .. note::
-    Output form Ansible orchestrator
+    Output form Cephadm orchestrator
 
-Decommission an OSD
+Remove an OSD
 -------------------
 ::
 
-    ceph orch osd rm <osd-id> [osd-id...]
+    orch osd rm <svc_id>... [--replace] [--force]
 
-Removes one or more OSDs from the cluster and the host, if the OSDs are marked as
-``destroyed``.
+Removes one or more OSDs from the cluster.
 
 Example::
 
     # ceph orch osd rm 4
```

Comment:
for Pacific, we have to correct the order to `ceph orch rm osd 4`

Reply:
do we? why is that?

Reply:
either we agree on 

```
ceph orch <verb> <objec> params
```
or on

```
ceph orch <objec>  <verb> params
```
Right now we have an evil mix of both.